### DISCUSSION TITLE
娶[ 老婆QQ号 | @老婆QQ ]等类似指令不可用。

### DISCUSSION THREAD

**Discussion by a2942** [FloatTech/ZeroBot-Plugin#242] [General]
试过很多次了，娶@QQ，或者 娶(QQ号) ，都没反应，使用的编译安装。

**Comment by fumiama** [comment]
有人遇到类似情况吗

**Reply by a2942** [reply]
在Linux上面进行编译安装和包括已经编译好直接下载安装的用都无法使用，包括你已经编译好的windows版本。

**Reply by a2942** [reply]
貌似我这里发生了一些网络性错误，发送该文本的时候似乎一次性发送了三四条。

**Reply by a2942** [reply]
对了，在使用"job"添加词库的时候发生了错误:
"ERROR:SQL logic error: no such table: 1na2gk3 (1)"
但是查看了目录文件夹，job.db数据库是存在的,我不明白他为什么要报错,并且机器人在发送图片的时候有大概率也会报错"ERROR:send image error",但是目标图片文件已经发送了的,如果有需要的话,你可以私自联系我来查看机器人运行。
